testing/src/test/java/io/stargate/it/cql/BoundStatementTest.java

```diff
@@ -24,10 +24,12 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.Isolated;
 
 /**
  * Note: all tests related to unset values assume that the protocol version is always 4 or higher.
  */
+@Isolated
```

Comment:
Class level causes way more problems than method level. With class level we have multiple classes all trying to start storage and stargate nodes at the same time.

Reply:
What were the problems?

I thought tests that use `@ClusterSpec(shared = true) @StargateSpec(shared = true)` do not restart the servers if they're already running, so I would expect to at least be able to parallelize all `BaseOsgiIntegrationTest`subclasses without any issue.

cc @dimas-b

Reply:
Some of them had port conflicts and others like BatchStatementTest have assertions about seed address (e.g. expecting 127.0.0.15 but is actually 127.0.0.18) that fails.

Reply:
Without extra changes, if a class with a different "spec" starts in parallel it will shut the share storage or Stargate down.

If we do not shutdown shared storage we'll get JMX port conflicts since `ccm` does not have user-level controls for JMX ports. `ccm` will assign JMX ports within the same cluster, but we can still get conflicts between different clusters.